locale/templates/LC_MESSAGES/server.pot

```diff
@@ -808,10 +836,10 @@ msgstr ""
 #: lib/senders/templates/subscriptionUpgrade.txt:3
```

Comment:
New string ID needed. Can consider change the number in the string ID.  https://github.com/mozilla-l10n/documentation/blob/main/src/localization/making_string_changes.md#why-is-it-necessary-to-use-new-ids